frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -106,7 +106,7 @@ function CriticalQuestions(props) {
   };
 
   const getGeocoding = () => {
-    const city = formValues[fields.CITY.key];
+    const city = formValues[fields.CITY.key] ? formValues[fields.CITY.key] : "";
```

Comment:
No need to change this line. If the user, does not input anything, city picker won't be triggered.